tests/event-bus/e2e-tester/e2e-tester.go

```diff
@@ -412,30 +424,30 @@ func checkSubscriberReceivedEventHeaders() error {
 			lowerResponseHeaders[strings.ToLower(k)] = resp[k]
 		}
 
-		if lowerResponseHeaders[sourceHeader][0] != srcID {
-			return fmt.Errorf("wrong response: %s, want: %s", lowerResponseHeaders[sourceHeader][0], srcID)
+		var testDataSets = []struct {
```

Comment:
Why do you need a new struct? Isn't is just key/value pairs? So a map could be a simply alternative.

Reply:
i was trying to follow this paradigm: https://github.com/golang/go/wiki/TableDrivenTests 

by having it in a struct i get compile time safety during iteration/